kube_scheduler/tests/test_sli_metrics.py

```diff
@@ -0,0 +1,87 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import mock
+import pytest
+import requests
+import requests_mock
+
+from datadog_checks.kube_scheduler import KubeSchedulerCheck
+
+instance = {'prometheus_url': 'http://localhost:10259/metrics'}
+
+# Constants
+CHECK_NAME = 'kube_scheduler'
+NAMESPACE = 'kube_scheduler'
+
+
+@pytest.fixture()
+def mock_metrics():
+    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'metrics_slis_1.27.3.txt')
+    with open(f_name, 'r') as f:
+        text_data = f.read()
+    with mock.patch(
+        'requests.get',
+        return_value=mock.MagicMock(
+            status_code=200, iter_lines=lambda **kwargs: text_data.split("\n"), headers={'Content-Type': "text/plain"}
+        ),
+    ):
+        yield
+
+
+def test_check_metrics_slis(aggregator, mock_metrics, mock_request):
+    mock_request.head('http://localhost:10259/metrics/slis', status_code=200)
+    c = KubeSchedulerCheck(CHECK_NAME, {}, [instance])
+    c.check(instance)
+
+    def assert_metric(name, **kwargs):
+        # Wrapper to keep assertions < 120 chars
+        aggregator.assert_metric(NAMESPACE + name, **kwargs)
```

Comment:
```suggestion
        aggregator.assert_metric(f"{NAMESPACE}.{name}", **kwargs)
```

Reply:
Since we run tests in Python 2 I couldn't use f-strings, but I used `format` instead